src/CoreWCF.Kafka/src/CoreWCF/Channels/KafkaMessageHeader.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace CoreWCF.Channels;
+
+public sealed class KafkaMessageHeader
+{
+    public KafkaMessageHeader(string key, byte[] value)
+    {
```

Comment:
Should you represent the Kafka body as byte[] or Memory\<byte\>? Maybe even ReadOnlyMemory\<byte\>?

Reply:
Done. I changed the KafkaMessageProperty.PartitionKey property type to ReadOnlyMemory<byte> too. 
Client side I kept byte[] as this the type we need to provide to Confluent.Kafka.